smart-contracts/contracts/Unlock.sol

```diff
@@ -433,9 +434,9 @@ contract Unlock is UnlockInitializable, UnlockOwnable {
             baseFee = 100;
           }
 
-          // tokensToDistribute is either == to the gas cost times 1.25 to cover the 20% dev cut
+          // tokensToDistribute is either == to the gas cost
           uint tokensToDistribute = ((estimatedGasForPurchase * baseFee) *
-            (125 * 10 ** 18)) /
+            (100 * 10 ** 18)) /
             100 /
```

Comment:
I think we can just remove that, and keep 10 ** 18 only ?

Reply:
yes... 🤦